packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -103,7 +106,31 @@ export function getValidatorApi({
   const MAX_API_CLOCK_DISPARITY_SEC = Math.min(0.5, config.SECONDS_PER_SLOT / 2);
   const MAX_API_CLOCK_DISPARITY_MS = MAX_API_CLOCK_DISPARITY_SEC * 1000;
 
-  /** Compute and cache the genesis block root */
+  async function waitForSlotWithDisparity(slot: number): Promise<void> {
```

Comment:
This seems to throw even if a past slot is passed as an argument. This is a change from the previous behavior.

Reply:
The naming "disparity" is confusing since its being used in subtlely different ways here vs in `currentEpochWithDisparity`.
Here its using `clock.isCurrentSlotGivenTolerance(.., 0, DISPARITY_MS)`
There its using `clock.isCurrentSlotGivenTolerance(.., DISPARITY_MS, 0)`

Reply:
Here we are sliding the `clock slot` to the future, in the epoch we are sliding it to the past. I think that was the original behavior. 

If we want to make it consistent please suggest.